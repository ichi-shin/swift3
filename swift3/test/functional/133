#!/bin/bash
# Copyright(c)2014 NTT corp. All Rights Reserved.
# DB1-1-8-1_12

. ./common

S3USER=tester

_s3_head /bucket | _filter_curl

_s3_post /bucket/object?uploads | _filter_curl xml
