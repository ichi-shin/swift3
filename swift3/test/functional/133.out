QA output created by 133
> s3curl --id tester -- -I -X HEAD http://SWIFT_HOST/bucket... 404
HTTP/1.1 404 Not Found
Content-Length: LENGTH
Content-Type: text/xml
Date: DATE
x-amz-id-2: TXID
x-amz-request-id: TXID
X-Trans-Id: TXID

> s3curl --id tester -- -X POST http://SWIFT_HOST/bucket/object?uploads... 404
<?xml version='1.0' encoding='UTF-8'?>
<Error>
  <Code>NoSuchBucket</Code>
  <Message>The specified bucket does not exist.</Message>
  <RequestId>TXID</RequestId>
  <BucketName>bucket</BucketName>
</Error>
